doc/wg/core-notes/core-2020-02-21.txt

```diff
@@ -0,0 +1,91 @@
+2/21/2020
+
+Attending: Phil, Alistair, Pat, Amit, Patrick, Garret, Sam, Holly, Branden, Hudson
+
+Johnathan is stuck in traffic.
+
+Amit: Let's start with updates. Who has updates?
+
+Amit: Hudson, can I volunteer you to talk about our call and your PR?
+
+Hudson: I put in a PR, then closed it, and put it back to my repo. I've been working on the scheduler interface, with an undergrad here and now with one of Amit's students. I had an interface that I was pretty happy with yesterday, but Amit pointed out that the scheduler might want to store processes in different data structures than just an array, e.g., in sets f queues. So the question is how I can make it so that rather than assuming an array, to have some kind of Container abstraction which a scheduler can specify. THis would also force us to make it so IPC no longer uses process array indices as a naming mechanism.
+
+Amit: Also, if you have thoughts on what would be a useful scheduler, please let us know. The design we come up with will be a compromise, based on the kinds of schedulers we expect being important. So know which is important.
+
+Hudson: I have an RR and a deadline based scheduler and a multilevel feedback queue scheduler, if there are others that people are interested in let me know.
+
+Amit: Johnathan is here now. Another update from Brad is that it looks like all of the necessary functionality and fixes to safely restart processes is done and waiting in PR. So that's a big deal if it's true. Let's move on to agenda items. Let's start with the more admin one: where do tests for the kernel from userspace liove?
```

Comment:
```suggestion
Amit: Johnathan is here now. Another update from Brad is that it looks like all of the necessary functionality and fixes to safely restart processes is done and waiting in PR. So that's a big deal if it's true. Let's move on to agenda items. Let's start with the more admin one: where do tests for the kernel from userspace live?
```